Library/Homebrew/test/test_integration_cmds.rb

```diff
@@ -834,4 +834,29 @@ class Testball < Formula
     desc_cache.unlink
     formula_file.unlink
   end
+
+  def test_style
+    foo_file = CoreTap.new.formula_dir/"foo.rb"
+    foo_file.write <<-EOS.undent
+      class Foo < Formula
+        url 'https://example.com/foo-1.0'
+      end
+    EOS
+
+    rubocop_yml = HOMEBREW_LIBRARY/".rubocop.yml"
+    rubocop_yml.write <<-EOS.undent
+      Style/Documentation:
+        Enabled: false
+
+      Style/StringLiterals:
+        EnforcedStyle: double_quotes
+    EOS
+
+    assert_match "Prefer double-quoted strings", cmd_fail("style")
+    assert_match "1 file inspected, 1 offense detected", cmd_fail("style", "#{foo_file}")
```

Comment:
I'm not sure what the problem and what the correct solution is, but `Bundler.with_clean_env` sadly isn't going to help (or change anything) here. It is already used in the implementation of `cmd[_fail]`, or to be more precise [in `cmd_output`](https://github.com/Homebrew/brew/blob/cab97cf4d782ce36762f0cd615ca59718f55b0d5/Library/Homebrew/test/test_integration_cmds.rb#L45-L66) where it is used to start a separate Homebrew process (but with the necessary adjustments for the testing environment) in which it then executes the specified Homebrew command.